tests/MongoDB.Driver.Tests/AggregateGraphLookupEnumerableFromOrToTests.cs

```diff
@@ -0,0 +1,241 @@
+﻿/* Copyright 2020-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System.Collections.Generic;
+using FluentAssertions;
+using MongoDB.Bson;
+using MongoDB.Driver.Core.Misc;
+using MongoDB.Driver.Core.TestHelpers.XunitExtensions;
+using Xunit;
+
+namespace MongoDB.Driver.Tests
+{
+    public class AggregateGraphLookupTests
+    {
+        // public methods
+        [SkippableFact]
+        public void GraphLookup_with_many_to_one_parameters_should_return_expected_result()
+        {
+            RequireServer.Check().Supports(Feature.AggregateGraphLookupStage);
+            var database = GetDatabase();
+            var collectionName = "collectionManyToOne";
+            EnsureTestDataManyToOne(database, collectionName);
```

Comment:
Nit: perhaps this and its companion methods could be declared as a local method to make the tests slightly easier to read?

Reply:
Perhaps... similar `EnsureTestData` methods in other classes are sometimes called from multiple test methods so a local method is not always possible.

It also might have the reverse effect, making the test harder to read by making the method longer?

Reply:
But I'd LGTM either way in this case depending on what others think.

Reply:
I feel like separate private methods are quite readable, so I leave it as is.